core/modules/admin_bar/css/admin_bar.css

```diff
@@ -6,6 +6,9 @@
  *
  * @see https://www.htmldog.com/articles/suckerfish
  */
+:root {
+  --admin-bar-height: 2.25em;
```

Comment:
We can't use "em" in this case, as any place, where the var is used may have a different value for that (inherited, or set). The means (according to my testing), the value's different depending on context.

Example: grab a heading from content, apply the var for positioning -> looks odd, as the heading has a bigger em value.

Reply:
We *could* switch to `rem`, but also in the whole admin-bar CSS, which requires a bigger change.

Reply:
The required change might not be that big, though, as currently everything inside admin-bar's already based on the base font size (12px).

So, if we made it:
font: normal .75rem "Lucida Grande", Verdana, sans-serif;

and then, based on that:
--admin-bar-height: 2.5rem;

... we'd use the same base unit for calculations...

Reply:
Thanks @indigoxela, insightful. I still haven't got a full grasp on em/rem.